gen/definitions/profile_parcels/policy_object_security_ips_signature.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Policy Object Security IPs Signature
```

Comment:
Policy Object Security IPS Signature List (it's IPS for Intrusion Prevention System that's why it should be all capital).